fastlane/lib/fastlane/commands_generator.rb

```diff
@@ -218,10 +218,11 @@ def run
       command :enable_auto_complete do |c|
         c.syntax = 'fastlane enable_auto_complete'
         c.description = 'Enable tab auto completion'
+        c.option '-c STRING[,STRING2]', '--custom STRING[,STRING2]', String, 'Add custom command(s) to use for auto complete'
```

Comment:
Could you expand a bit on this comment? From this description it sounds a bit unclear to me what this does

Reply:
> Add custom command(s) for which tab auto complete should work too

Reply:
Yup, sounds better to me! Maybe `should be enabled too` instead of `should work too`